index.html

```diff
@@ -418,39 +415,21 @@ <h2>The <dfn>IdleDeadline</dfn> interface</h2>
     <h2>Processing</h2>
     <section>
       <h2>Start an idle period algorithm</h2>
-      <p>The <dfn>start an idle period algorithm</dfn>, which is called
-      by the <a data-cite="html#event-loop-processing-model">event loop processing model</a> when it determines that
-      the <a>event loop</a> is otherwise idle:</p>
+      <p>To <dfn export>start an idle period</dfn> given {{Window}} <var>window</var> and
+      <var>getDeadline</var>, an algorithm that returns a {{DOMHighResTimeStamp}}: [[HR-TIME]]
```

Comment:
The reading is a little bit confusing... the type usually comes first:

```suggestion
      an algorithm that returns a {{DOMHighResTimeStamp}} <var>getDeadline</var>: [[HR-TIME]]
```